lib/questionnaires/view/src/questionnaire_theme.dart

```diff
@@ -45,6 +45,12 @@ class QuestionnaireThemeData {
   /// Returns whether the score is displayed while filling (in stepper mode only)
   final bool showScore;
 
+  /// Returns whether the validation must be done before moving to next page (in stepper mode only)
+  final bool validatePerPage;
+
+  /// Returns whether to automatically jump to the next page after an answer is selected (in stepper mode only)
+  final bool autoJumpNextPage;
+
```

Comment:
After thinking about this again, only `autoJumpNextPage` is necessary for our internal purpose, do you think it's better to remove `validatePerPage` or it's fine to retain it since it might be necessary for the community?